update-rules.json

```diff
@@ -829,5 +828,286 @@
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+   },
+   "ynab": {
+    "url": "http://classic.youneedabudget.com/download",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='_Setup.exe']"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+   },
+   "veracrypt": {
+    "url": "https://sourceforge.net/projects/veracrypt/files/",
+    "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+        "selector": "#files > div.download-bar",
+		"baselink": "https://sourceforge.net/projects/veracrypt/files/VeraCrypt%20{{.version}}/VeraCrypt%20Setup%20{{version}}.exe/download"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"  
+	},
+	"openssh": {
+    "url": "https://www.mls-software.com/opensshd.html",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='.exe']"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+   },
+	"sublime-text-dev": {
+    "url": "https://www.sublimetext.com/3",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "#dl_win_32 > a:nth-child(1)"
+      },
+      "x86_64":{
+        "rule_type": "css-link",
+        "selector": "#dl_win_64 > a:nth-child(1)"
+      }
+    },
+    "version_extractor": "Sublime Text Build ([0-9]*)(?: x64)? Setup\\.exe$"
+  },
+  	"sublime-text": {
+    "url": "https://www.sublimetext.com/2",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "#dl_win_32 > a:nth-child(1)"
+      },
+      "x86_64":{
+        "rule_type": "css-link",
+        "selector": "#dl_win_64 > a:nth-child(1)"
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d{1,5}[a-z])"
+  },
+    "subversion": {
+    "url": "https://sliksvn.com/download/",
+    "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+		"baselink": "https://sliksvn.com/pub/Slik-Subversion-{{.version}}-win32.msi",
+        "selector": "div.actions"
+      },
+      "x86_64":{
+        "rule_type": "css-html-version",
+		"baselink": "https://sliksvn.com/pub/Slik-Subversion-{{.version}}-x64.msi",
+        "selector": "div.actions"
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d){1,5}"
+  },
+    "dependency-walker": {
+    "url": "http://www.dependencywalker.com",
+    "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+        "selector": "body > font > div",
+		"baselink": "http://www.dependencywalker.com/depends{{.version#}}_x86.zip"
+      },
+      "x86_64":{
+        "rule_type": "css-html-version",
+        "selector": "body > font > div",
+		"baselink": "http://www.dependencywalker.com/depends{{.version#}}_x64.zip"
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d){1,5}"
+  },
+    "emacs": {
+    "url": "http://ftp.gnu.org/gnu/emacs/windows/",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='i686-w64-mingw32.zip']"
```

Comment:
I'm not sure I understand their naming convention. It seems to vary wildly from a release to the next. Do you think this rule will be able to match future releases?